src/sqlancer/mariadb/MariaDBProvider.java

```diff
@@ -77,6 +79,9 @@ public void generateDatabase(MariaDBGlobalState globalState) throws Exception {
             case CREATE_INDEX:
                 nrPerformed = globalState.getRandomly().getInteger(0, 2);
                 break;
+            case DELETE:
```

Comment:
Perhaps we could also generate them with a low probability? This logic will include 10 `DELETE` statements into every test case, rather than probabilistically generate some for some test cases, so tables are likely to be empty for many test cases.